scripts/systemd/prox-update-gcals.service

```diff
@@ -5,7 +5,7 @@ Description=prox update gcals
 Type=simple
 WorkingDirectory=/home/prox/prox-server
 Environment=PYTHONPATH=/home/prox/prox-server
-ExecStart=/home/prox/prox-server/env/bin/python scripts/update_gcals.py
+ExecStart=/home/prox/prox-server/env/bin/python -m scripts.update_gcals
```

Comment:
Great - if we're using just scripts/update_gcals.py, I should be able to remove the update_gcals call from main.py __init__, that still calls the updateGcals method.